contracts/examples/multisig/interact-rs/src/multisig_interact_nfts.rs

```diff
@@ -32,196 +32,153 @@ impl MultisigInteract {
+            .interactor
+            .sc_call_get_result(
+                ScCallStep::new()
+                    .call(self.state.multisig().propose_async_call(
+                        system_sc_address,
+                        ISSUE_COST,
+                        "registerAndSetAllRoles".to_string(),
+                        MultiValueVec::from([
+                            COLLECTION_NAME.as_bytes(),
+                            COLLECTION_TICKER.as_bytes(),
+                            TOKEN_TYPE.as_bytes(),
+                            top_encode_to_vec_u8_or_panic(&0u32).as_slice(),
+                        ]),
+                    ))
+                    .from(&self.wallet_address)
+                    .gas_limit("10,000,000"),
             )
-            .into_blockchain_call()
-            .from(&self.wallet_address)
-            .gas_limit("10,000,000");
-
-        self.interactor.sc_call(&mut typed_sc_call).await;
-
-        let result = typed_sc_call.result();
-        if result.is_err() {
-            println!(
-                "propose issue collection with roles failed with: {}",
-                result.err().unwrap()
-            );
-            return None;
-        }
+            .await;
 
-        let action_id = result.unwrap();
-        println!("successfully proposed issue colllection with roles action `{action_id}`");
-        Some(action_id)
+        println!("successfully proposed issue colllection with roles all action `{action_id}`");
+        action_id
     }
 
     pub async fn issue_collection_with_all_roles(&mut self) {
         println!("proposing issue collection with all roles...");
         let action_id = self.propose_issue_collection_with_all_roles().await;
-        if action_id.is_none() {
-            return;
-        }
 
-        let action_id = action_id.unwrap();
         println!("perfoming issue collection with all roles action `{action_id}`...");
 
         if !self.quorum_reached(action_id).await && !self.sign(action_id).await {
             return;
         }
         println!("quorum reached for action `{action_id}`");
 
-        let mut typed_sc_call = self
-            .state
-            .multisig()
-            .perform_action_endpoint(action_id)
-            .into_blockchain_call()
-            .from(&self.wallet_address)
-            .gas_limit("80,000,000");
-
-        self.interactor.sc_call(&mut typed_sc_call).await;
-
-        let new_issued_token_identifier =
-            typed_sc_call.response().new_issued_token_identifier.clone();
-
-        if let Some(token_identifier) = new_issued_token_identifier {
-            self.collection_token_identifier = token_identifier;
-            println!(
-                "collection token identifier: {}",
-                self.collection_token_identifier
-            );
-            return;
-        }
-
-        println!("perform issue collection with all roles failed");
+        self.interactor
+            .sc_call_use_raw_response(
```

Comment:
Should use `sc_call_use_result`, no need to go for the raw response, or `sc_call_get_result`